docroot/profiles/custom/cgov_site/modules/custom/pdq_cancer_information_summary/src/OrphanCleanup.php

```diff
@@ -123,7 +123,6 @@ public function dropOrphanedSummarySections($max_deletions = 0) {
       }
       $results = $query->execute();
       foreach ($results as $result) {
-        $entity = $storage->load($paragraph_id);
```

Comment:
@bkline Does this removal make sense? This is just setting a var that is not used, but who knows if you found that `$storage->load()` did some sort of side effect for the delete to work. (Although I see no comment as such, and you usually comment the odd things)